cmd/cnab-run/main.go

```diff
@@ -13,6 +13,7 @@ const (
 	cnabActionUninstall = cnabAction("uninstall")
 	cnabActionUpgrade   = cnabAction("upgrade")
 	cnabActionStatus    = cnabAction("status")
+	cnabActionInspect   = cnabAction("inspect")
```

Comment:
There was discussion of namespacing custom actions, did we decide not to?

Reply:
Oh right! 🤦‍♂️

Reply:
I'll do that in another PR following-up, as it will also modify the `status` custom action.